### PR TITLE
Update the log forwarding docs

### PR DESCRIPTION
**Pull request by ghost**
https://issues.newrelic.com/browse/NR-108281

### CODE REVIEW

File: src/content/docs/logs/forward-logs/overview of log-forwarding.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  - /docs/logs/enable-log-management-in-new-relic/enable-log-monitoring-in-new-relicYES
+  - /docs/logs/forward-logs/enable-log-management-new-relicfor
+  - /docs/logs/forward-logs/enable-log-management-new-relic
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
+---
+
+Log forwarding is a way to collect and send your host logs so you can have all your logs in one place. By default, our APM agents collect application-level logs. But if you want to send host logs, you can set up log forwarding depending on what kind of host you have. 
+
+We offer many options to collect and send host logs, which are listed in the table below. Basically, you can use one of our log fowarding options (i.e., our infrastructure agent, Kubernetes plugin, or log API) or a third-party log forwarder (i.e., AWS, Azure, GCP).
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/overview of log-forwarding.mdx:29]:
```suggestion
We offer many options to forward host logs, which are listed in the table below. Basically, you can use one of our log fowarding options (i.e., our infrastructure agent, Kubernetes plugin, or log API) or a third-party log forwarder (i.e., AWS, Azure, GCP).
```

File: src/content/docs/logs/forward-logs/overview of log-forwarding.mdx [hunk 2]
```diff
@@ -0,0 +1,122 @@
+---
+title: Forward your logs to New Relic
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/overview of log-forwarding.mdx:2]:
```suggestion
title: Forward host logs to New Relic
```

File: src/content/docs/logs/forward-logs/overview of log-forwarding.mdx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    
+    <tr>
+      <td>
+        Docker/Kubernetes containers
+      </td>
+      <td>
+        For Kubernetes environments, use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/).
+        
+        For Docker environments, we offer a few options:
+        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/overview of log-forwarding.mdx:85]: @lchapman4 @marcelschlapfer I want to move these three bullet points over to the "Forward logs with infra agent" page, as a new section that can be linked to from this table. But for this first bullet point, does this mean APM logs in context automatically collects Docker logs? Or does it mean in conjunction with the infra agent?

File: src/content/docs/logs/forward-logs/overview of log-forwarding.mdx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      <td>
+        * [Our log API](/docs/logs/log-api/introduction-log-api/)
+        * Akamai DataStream 2: 
+          * [Akamai instructions](https://techdocs.akamai.com/datastream2/docs/stream-new-relic)
+          * [Quickstart](https://newrelic.com/instant-observability/datastream2/cb995039-39d8-44d9-97c3-58174a1337f2)
+        * Fastly: 
+          * [Fastly instructions](https://docs.fastly.com/en/guides/log-streaming-newrelic-logs)
+          * [Quickstart](https://newrelic.com/instant-observability/fastly-cdn/c5c5dd30-dcdf-46b6-9412-f9a1bba5a600)
+        * [Vector plugin](/docs/logs/forward-logs/vector-output-sink-log-forwarding)
+        * Use the OpenTelemetry SDK to send logs from your apps to an OpenTelemetry collector, which can forward them to New Relic via OTLP.
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/overview of log-forwarding.mdx:118]: @lchapman4 @marcelschlapfer I'd like to add a docs link with instructions to set this up, but I'm not sure which page for specifically forwarding logs. Can you help me sourcing the best docs link?

File: src/content/docs/logs/forward-logs/overview of log-forwarding.mdx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        
+        For Docker environments, we offer a few options:
+        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
+        * Use our infrastructure agent. In the `logging.yml` config file be sure to tail the `/var/log/containers/*.log` directory. For example:
+          ```yml
+          logs:
+           - name: Docker-logs
+             file: /var/log/containers/*.log # Path to Docker log files
+          ```
+        * Use our [Standalone Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output).
```
[the remark this answers is not in the document]
Comment (ally-sassman) [reply src/content/docs/logs/forward-logs/overview of log-forwarding.mdx:92]: @lchapman4 @marcelschlapfer I want to move these three Docker bullet points over to the "Forward logs with infra agent" page as a new section dedicated to using the infra agent to forward Docker logs. But for this first bullet point, does this mean APM logs in context automatically collects/displays Docker logs? Or does it logs in context data is collected in conjunction with the infra agent?

File: src/content/docs/logs/forward-logs/overview of log-forwarding.mdx [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+      <td>
+        * [Our log API](/docs/logs/log-api/introduction-log-api/)
+        * Akamai DataStream 2: 
+          * [Akamai instructions](https://techdocs.akamai.com/datastream2/docs/stream-new-relic)
+          * [Quickstart](https://newrelic.com/instant-observability/datastream2/cb995039-39d8-44d9-97c3-58174a1337f2)
+        * Fastly: 
+          * [Fastly instructions](https://docs.fastly.com/en/guides/log-streaming-newrelic-logs)
+          * [Quickstart](https://newrelic.com/instant-observability/fastly-cdn/c5c5dd30-dcdf-46b6-9412-f9a1bba5a600)
+        * [Vector plugin](/docs/logs/forward-logs/vector-output-sink-log-forwarding)
+        * Use the OpenTelemetry SDK to send logs from your apps to an OpenTelemetry collector, which can forward them to New Relic via OTLP.
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/overview of log-forwarding.mdx:118]:
```suggestion
        * [Use the OpenTelemetry SDK](add docs link) to send logs from your apps to an OpenTelemetry collector, which can forward them to New Relic via OTLP.
```

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 7]
```diff
@@ -659,7 +643,19 @@ Here is an example of a `logging.d/` configuration file in YAML format. For more
...
+If your app is running in a Docker container, be sure tail the `/var/log/containers/*.log` directory for `Docker-logs`, which tells our logging system to continuously monitor the `/var/log/containers/*.log` directory for new log entries.
+
+#### Example
+
+          ```yml
+          logs:
+           - name: Docker-logs
+             file: /var/log/containers/*.log # Path to Docker log files
+          ```
+
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:657]: @lchapman4 @marcelschlapfer What do you think of this addition?

File: src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx [hunk 8]
```diff
@@ -659,7 +643,19 @@ Here is an example of a `logging.d/` configuration file in YAML format. For more
   </Collapser>
 </CollapserGroup>
 
-### Step 4. View your log data [#find-data]
+### Step 4: (Optional) Collect Docker host logs
```
Comment (ally-sassman) [inline src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx:646]:
```suggestion
### Step 4: (Optional) Collect Docker host logs [#docker]
```

File: src/content/docs/logs/forward-logs/overview of log-forwarding.mdx (hunk 4 above)
Comment (lchapman4) [reply src/content/docs/logs/forward-logs/overview of log-forwarding.mdx:118]: Maybe use the link to our infra forwarder? https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/

Comment (ally-sassman) [reply src/content/docs/logs/forward-logs/overview of log-forwarding.mdx:118]: But this bullet point is for OpenTelemetry, and we don't have any OTEL info on https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/ :(

**Pull request closed without merging**